### PR TITLE
[ja] Localize glossary entry for "LimitRange"

### PR DESCRIPTION
**Pull request by leaf-soba**
### Description
- I find this issue by `good first issue` label know this issue and want to contribute to this repo.
- I know this issue already have a assignee and a PR, since old PR didn't update to follow the guide in two weeks, so I think I can take this issue to make a new PR. Sorry I'm new in Github and this repo, I don't know this behavior is polite or not, if I have done something wrong please tell me I'll fix it.

### Issue

Closes: #47684

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle [ja] Localize glossary entry for "LimitRange"

File: content/ja/docs/reference/glossary/limitrange.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: LimitRange
+id: limitrange
+date: 2019-04-15
+full_link:  /docs/concepts/policy/limit-range/
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/limitrange.md:5]:
```suggestion
full_link:  /ja/docs/concepts/policy/limit-range/
```

https://kubernetes.io/ja/docs/contribute/localization/#basic-policy

> すでに日本語訳が存在するページにリンクを張る場合は、/ja/を含めたURLを使用する

File: content/ja/docs/reference/glossary/limitrange.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+short_description: >
+  各ネームスペース内のコンテナまたはポッドごとのリソース消費量に制限するための制約を提供します。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/limitrange.md:7]:
```suggestion
  各Namespace内のコンテナまたはPodごとのリソース消費量を制限するための制約を提供します。
```

FYI: https://kubernetes.io/ja/docs/contribute/localization/#terminology

> Kubernetesのリソース名や技術用語などは、原則としてそのままの表記を使用します。 例えば、PodやService、Deploymentなどは翻訳せずにそのまま表記してください。

File: content/ja/docs/reference/glossary/limitrange.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+tags:
+- core-object
+- fundamental
+- architecture
+related:
+ - pod
+ - container
+
+---
+ 各ネームスペース内の{{< glossary_tooltip text="コンテナ" term_id="container" >}}または{{< glossary_tooltip text="ポッド" term_id="pod" >}}ごとのリソース消費量に制限するための制約を提供します。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/limitrange.md:19]:
```suggestion
 各Namespace内の{{< glossary_tooltip text="コンテナ" term_id="container" >}}または{{< glossary_tooltip text="Pod" term_id="pod" >}}ごとのリソース消費量を制限するための制約を提供します。
```

File: content/ja/docs/reference/glossary/limitrange.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+<!--more--> 
+LimitRangeは、タイプごとに作成できるオブジェクトの数量を制限するだけでなく、ネームスペース内の個々の{{< glossary_tooltip text="コンテナ" term_id="container" >}}または{{< glossary_tooltip text="ポッド" term_id="pod" >}}によって要求または消費される計算リソースの量も制限します。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/limitrange.md:22]:
```suggestion
LimitRangeは、タイプごとに作成できるオブジェクトの数量を制限するだけでなく、Namespace内の個々の{{< glossary_tooltip text="コンテナ" term_id="container" >}}または{{< glossary_tooltip text="Pod" term_id="pod" >}}によって要求または消費される計算リソースの量も制限します。
```

**Review by Okabe-Junya — COMMENTED** [review]
Thank you for your interest in SIG-Docs Japanese localization project!

I appreciate that you found a good first issue and started working on it. Please note that I had indicated “Note that this issue is for Japanese Speaker” in the description. 
I am not sure what languages you are proficient in, but in l10n, please make sure to contribute to the languages you are well-versed in, Thanks!

File: content/ja/docs/reference/glossary/limitrange.md (hunk 2 above)
Comment (leaf-soba) [reply content/ja/docs/reference/glossary/limitrange.md:7]: I saw the quote content in the contribute guide before, but I also found a lot of `ポッド and `ネームスペース` in this repo, such as `/content/ja/docs/reference/kubectl/cheatsheet.md`. It confuses me, now I know I need to follow the guide, thanks for advice, I'll fix it.

**Comment by leaf-soba** [comment]
> Thank you for your interest in SIG-Docs Japanese localization project!
> 
> I appreciate that you found a good first issue and started working on it. Please note that I had indicated “Note that this issue is for Japanese Speaker” in the description. I am not sure what languages you are proficient in, but in l10n, please make sure to contribute to the languages you are well-versed in, Thanks!

Thanks for replying! Although I'm not native speaker in Japanese, but I passed JLPT N1 and worked in Japan about 3 years. I'll do my best to contribute to this repo.

**Review by Okabe-Junya — COMMENTED** [review]
Thanks!

/lgtm
/label tide/merge-method-squash
/assign bells17 
for approval

**Review by b1gb4by — APPROVED** [review]

**Review by Okabe-Junya — APPROVED** [review]
Sorry for delay in response, but we got 2 lgtm so let me merge as a Japanese l10n owner

/lgtm 
/approve

**Pull request merged**